### PR TITLE
Create distribution badges to mark features specific to a distribution

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

Add badges for the different The Things Stack distributions. Looks like this, with a tooltip on hover:

Screenshot 2021-01-13 at 12 45 49
Screenshot 2021-01-13 at 11 54 28
Screenshot 2021-01-13 at 12 47 56

Clicking the badges takes you to a page that explains what they are:

Screenshot 2021-01-13 at 11 38 33

@pierrephz, PR needs a TTN frame. Is this a colored square or the TTN logo? @johanstokking ?

#### Changes:

- Add distribution icons
- Remove `distributions-inline` shortcode (only distributions shortcode exists now, always produces the icons)
- Rename Marketplace launcher to AWS launcher
- Rename Community to The Things Network

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/themes/the-things-stack/layouts/partials/distribution-badges.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<div class="buttons">
+	{{ range . }}
+		{{ $distribution := index site.Data.distributions . }}
+		<a href="/download" class="button is-small is-short" style="color:{{ index $distribution "color" }};border-color:{{ index $distribution "color" }};">
+			<span class="icon">
+		    	<img src="{{ index $distribution "icon" }}"></img>
+		    </span>
+		    <span class="has-text-weight-bold">{{ . }}</span>
```
Comment (htdvisser) [inline doc/themes/the-things-stack/layouts/partials/distribution-badges.html:8]: - Inconsistent use of tabs and spaces
- Use `relURL` (like we do in check-icon: `src="{{ "img/check.svg" | relURL }}"`)

File: doc/themes/the-things-stack/layouts/_default/single.html [hunk 2]
```diff
@@ -7,18 +7,21 @@ <h1 class="title is-size-2">{{ .Title }}
...
-    {{ $distributions = slice $distributions }}
-  {{ end }}
-  {{ partial "distributions" (dict "distributions" $distributions) }}
+	{{ $distributions := . }}
+	{{ if ( not ( reflect.IsSlice . ) ) }}
+		{{ $distributions = slice $distributions }}
+  	{{ end }}
+  	<p class="subtitle">
+  		{{ partial "distributions" (dict "distributions" $distributions "header" true ) }}
+	</p>
```
Comment (htdvisser) [inline doc/themes/the-things-stack/layouts/_default/single.html:17]: Inconsistent use of tabs and spaces

File: doc/themes/the-things-stack/layouts/_default/list.html [hunk 3]
```diff
@@ -7,15 +7,17 @@ <h1 class="title is-size-2">{{ .Title }}
...
-    {{ $distributions = slice $distributions }}
-  {{ end }}
-  {{ partial "distributions" (dict "distributions" $distributions) }}
+	{{ $distributions := . }}
+	{{ if ( not ( reflect.IsSlice . ) ) }}
+		{{ $distributions = slice $distributions }}
+  	{{ end }}
+  	<p class="subtitle">
+  		{{ partial "distributions" (dict "distributions" $distributions "header" true ) }}
+	</p>
```
Comment (htdvisser) [inline doc/themes/the-things-stack/layouts/_default/list.html:17]: Inconsistent use of tabs and spaces

File: doc/themes/the-things-stack/assets/css/theme.scss [hunk 4]
```diff
@@ -64,60 +68,10 @@ $table-cell-padding: 1.2em 0.75em;
...
-@import "bulma/sass/grid/columns.sass";
-// @import "bulma/sass/grid/tiles.sass";
-
-// @import "bulma/sass/layout/_all";
-@import "bulma/sass/layout/hero.sass";
-@import "bulma/sass/layout/section.sass";
-@import "bulma/sass/layout/footer.sass";
+// Modular imports broken in bulma 0.9
+// https://github.com/jgthms/bulma/issues/2797
+
```
Comment (htdvisser) [inline doc/themes/the-things-stack/assets/css/theme.scss:73]: I'm fine with removing this comment and just importing the entire library

**Review by pierrephz — CHANGES_REQUESTED** [review]
be careful with the badges
- don't use ITC avant-garde here

image

- make sure the balance of the badge is right (icon + text centered in the frame)

image

- make a bit more space between the icon and the text in the badge (see below)

Screenshot 2020-12-02 at 11 03 45

**Comment by pierrephz** [comment]
> I also left the inline distribution shortcode untouched because sometimes it has to fit in tiny tables, but @pierrephz maybe you can think of an aesthetic way to integrate it here?:

regarding this, I am thinking about an. "information" label that you can hover to get the info (see screenshot below):

image

**Comment by benolayinka** [comment]
> > I also left the inline distribution shortcode untouched because sometimes it has to fit in tiny tables, but @pierrephz maybe you can think of an aesthetic way to integrate it here?:
> 
> regarding this, I am thinking about an "information" label that you can hover to get the info (see screenshot below):
> 
> image

I like this idea, but maybe we use a tiny colored stack of icons instead of the bubble? @htdvisser also suggested wrapping sections to fix https://github.com/TheThingsIndustries/lorawan-stack-docs/issues/139

**Comment by benolayinka** [comment]
@pierrephz @kschiffer how about something like this for inline? we could also even minimize all the badges

Screenshot 2020-12-28 at 17 16 04

**Comment by benolayinka** [comment]
and for the badges @pierrephz ? or we can just change everything to the stacked icons

Screenshot 2020-12-28 at 17 31 05

**Comment by kschiffer** [comment]
> @pierrephz @kschiffer how about something like this for inline? we could also even minimize all the badges
> 
> Screenshot 2020-12-28 at 17 16 04

Just the (stacked) icons is too implicit. I think this only works in conjunction with a descriptive label, e.g. `All deployment models`.

**Comment by pierrephz** [comment]
Yea could be
What is the purpose? save some space?

> Actually, I think for inline, this can work, if it shows the tooltip on hover. I still think the stacked icon needs to be a bit clearer though.

yes, should be a bit larger, and more space in-between if we go for this

**Comment by benolayinka** [comment]
> Yea could be
> What is the purpose? save some space?
> 
> > Actually, I think for inline, this can work, if it shows the tooltip on hover. I still think the stacked icon needs to be a bit clearer though.
> 
> yes, should be a bit larger, and more space in-between if we go for this

Yep, especially for places with no space. It is also much less intrusive.

Screenshot 2020-12-29 at 12 56 27

**Comment by benolayinka** [comment]
ok @pierrephz here's what I have today:

On cards:
Screenshot 2020-12-29 at 19 43 33

Inside page and inline:
Screenshot 2020-12-29 at 19 43 49

@kschiffer the frustrating issue i'm encountering is that the image span isn't lining up with the text. There's no up or down margin or anything on the spans, I can't understand why they are moving vertically?

Screenshot 2020-12-29 at 19 38 50

Screenshot 2020-12-29 at 19 38 11

**Comment by benolayinka** [comment]
Decided to go with the stacked icons because it's clean, simple, fits everywhere, and is nearly fixed size. @pierrephz please take a look. Also, we need an icon for The Things Network. Maybe a little cloud or something?

**Review by pierrephz — APPROVED** [review]
@benolayinka that works
find here the cloud icon for ttn: 
[ttn.svg.zip](https://github.com/TheThingsIndustries/lorawan-stack-docs/files/5809071/ttn.svg.zip)

**Review by nejraselimovic — APPROVED** [review]
Looks great, good job! Here's some remarks:

- I had to change `@import "@creativebulma/bulma-tooltip/dist/bulma-tooltip.min"` to `@import "@creativebulma/bulma-tooltip/dist/bulma-tooltip.min.css"` in order to make this work
- I'm not seeing tooltip on hover

**Review by kschiffer — CHANGES_REQUESTED** [review]
> Looks great, good job! Here are some remarks:
> 
> *   I had to change `@import "@creativebulma/bulma-tooltip/dist/bulma-tooltip.min"` to `@import "@creativebulma/bulma-tooltip/dist/bulma-tooltip.min.css"` in order to make this work
> *   I'm not seeing tooltip on hover

Same here.

**Comment by benolayinka** [comment]
> Looks great, good job! Here are some remarks:
> 
> * I had to change `@import "@creativebulma/bulma-tooltip/dist/bulma-tooltip.min"` to `@import "@creativebulma/bulma-tooltip/dist/bulma-tooltip.min.css"` in order to make this work
> * I'm not seeing tooltip on hover

Should be fixed now, the I added the `bulma-tooltip` package with npm when I should have used yarn, so it wasn't in `yarn.lock` and `make deps` didn't install it for you.

**Comment by benolayinka** [comment]
> @benolayinka that works
> find here the cloud icon for ttn:
> [ttn.svg.zip](https://github.com/TheThingsIndustries/lorawan-stack-docs/files/5809071/ttn.svg.zip)

With the ttn frame it looks like this:

Bildschirmfoto 2021-02-03 um 13 59 46

Is that okay or we do something else?

**Review by kschiffer — APPROVED** [review]

**Review by neoaggelos — APPROVED** [review]

**Review by adriansmares — APPROVED** [review]

**Review by johanstokking — APPROVED** [review]
Code owned files LGTM

**Pull request merged**